Shared/ThresholdKeyView.swift

```diff
@@ -65,7 +66,10 @@ struct ThresholdKeyView: View {
                                 }
                                 
                                 // TODO: Add proper messages, 1/2 and 2/2 (old accounts, new accounts)
-                                alertContent = "\(totalShares) shares created"
+                                alertContent = "\(totalShares - required_share)/\(totalShares) shares created."
+                                if totalShares == 2 && required_share == 1 {
```

Comment:
No, comparison is incorrect, if total_shares < required_shares. 

This doesn't take into account the device share being added.

Reply:
fixed